source/includes/_evaluation-api.md

```diff
@@ -0,0 +1,37 @@
+# Evaluation API
+```shell
+curl -X POST https://collection-prod.inspcdn.net/evaluation \
+    -H "Authorization: Bearer $API_KEY" \
+    -H "Content-Type: application/json" \
+    -d '{
+        "sale_id": "12345",
+        "account_id": "157421",
+        "sale_datetime": 1579792758,
+        "event_date_id": "23553",
+        "sale_total_value": 54.26,
+        "first_six_digits_cc": "455326",
+        "last_four_digits_cc": "0012",
+        "holder_cpf": "741.112.235-53"
+    }'
+```
+
+Our evaluation API consists of a single resource: `/evaluation`. This resource should get used whenever you have a Sale that you would like to get evaluated by Inspetor.
+
+It is a `POST` endpoint, and the fields it accepts are as follows:
+
+Property                        | Type    | Description
+--------                        | ----    | -----------
+sale_id                         | String  | The sale ID
```

Comment:
Nit: What do you think of using the same description of the `id` in the `sales` endpoint? `The unique identifier for the sale within your platform`